### DISCUSSION TITLE
Not a function error

### DISCUSSION THREAD

**Discussion by ThreeSpaceTab** [JaapvanEkris/openrowingmonitor#10] [General]
After doing the install and running
`sudo systemctl status openrowingmonitor`
I get the error
pigpio.gpioCfgClock is not a function.

Now, while I have the display metrics on screen, no data is shown when rowing, and the session logs do not get created.

**Comment by JaapvanEkris** [comment]
Did you run the instll script correctly. You need to make sure the pigpio package is installed both as an apt-get package and as an npm.

**Reply by ThreeSpaceTab** [reply]
I ran the install script and pressed Y for all options. I fixed it by going through the steps manually.
